scripts/src/main/resources/scripts/command/aws

```diff
@@ -17,16 +17,28 @@ source "$(dirname "${0}")"/../functions
 # Call aws with specified arguments.
 function doRun() {
   doSetup silent
-  doRunCommand "aws ${*}"
+  if doIsWindows
+  then
+    doRunCommand "\"/c/Program Files/Amazon/AWSCLIV2/aws\" ${*}"
```

Comment:
This is redundant to line 37 and as @tobka777 mentioned you should use `%ProgramFiles%` variable as on some windows systems it may be on a different drive or in a different folder.
Unfortunately this variable is not directly available but I figured out that you can get it in git-bash via `cygpath "$(powershell  -c '${env:ProgramFiles}')"`.

I would suggest to do the following:
```
function findAws() {
  AWS="aws"
  AWS_INSTALLED=
  if command -v aws &> /dev/null
  then
    AWS_INSTALLED="yes"
  elif doIsWindows
  then
    local aws_win
    aws_win="$(powershell  -c '${env:ProgramFiles}')"
    aws_win="$(cygpath "${aws_win}")"
    aws_win="${aws_win}/Amazon/AWSCLIV2/aws"
    if [ -x "${aws_win}" ]
    then
      AWS="${aws_win}"
      AWS_INSTALLED="yes"
    fi
  fi
}
```

Then you can call this function before invoking `aws` and to run it simply do:
```
doRunCommand "'${AWS}' ${*}"
```

Reply:
@CREITZ25 Great. Thanks that you found this `${PROGRAMFILES}` variable.
I tested this and it works.
No idea, why I did not get this with tab completion when I searched for this during my review.
Lets cross fingers and hope for the best.
If we or someone else can ever reproduce this newline instead of whitespace error, we simply create a new bug issue and try to fix or workaround it.